src/UNL/UCBCN/Manager/Calendar.php

```diff
@@ -38,4 +43,28 @@ public function getCategorizedEvents() {
         return $categories;
     }
 
+    private function archiveEvents() {
+        # find all posted (upcoming) events on the calendar
+        $events = $this->calendar->getEvents('posted');
+
+        # check each event to see if it has passed
+        foreach ($events as $event) {
+            # we will consider it passed if one datetime is in the past
+            # (this is how the previous system did it)
```

Comment:
OK, I had thought of doing it that way because it seems better, but went with current implementation. Will switch it over